R/deltaccd.R

```diff
@@ -147,39 +157,44 @@ calcCCD = function(refCor, emat, groupVec = NULL, refEmat = NULL, nPerm = 1000,
   nComb = choose(nrow(emat), length(geneNames))
 
   if (nPerm>1) {
+    
     result = foreach(groupNow = sort(unique(groupVec)), .combine = rbind) %do% {
```

Comment:
Not going to do this for all, but just like with `if` statements, don't we add a space after `foreach` @jakejh ? I could be wrong.

Reply:
Resolving because I was mistaken.